example/synopsis.pl

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+
+use Term::EditLine qw(CC_EOF);
+
+use v5.14;
```

Comment:
I am not sure why 5.14 is necessary?

Reply:
Well, it's modern Perl, but in this case it was just to use "say". Can be eliminated.

Reply:
To use say we'd only need `use v5.10`.

nit: I'd prefer any `use <perl version>` statement to go immediately after the `use warnings`.

```perl
use strict;
use warnings;
use 5.010;
```